server/services/professionalContactsService.ts

```diff
@@ -160,6 +176,115 @@ export default class ProfessionalContactsService {
 
     return sortByDateTime(right.startDate, left.startDate) // Most recently added first
   }
+
+  getProfessionalContactsOverview(
+    clientToken: string,
+    { prisonId, bookingId, prisonerNumber }: Prisoner,
+    apiErrorCallback: (error: Error) => void = () => null,
+  ): Promise<YouthStaffContacts | StaffContacts> {
+    const isYouthPrisoner = youthEstatePrisons.includes(prisonId)
+    return isYouthPrisoner
+      ? this.getYouthStaffContactsOverview(clientToken, bookingId)
+      : this.getStaffContactsOverview(clientToken, bookingId, prisonerNumber, prisonId, apiErrorCallback)
+  }
```

Comment:
I'm sure we can find a better way of adding a banner than having to prop drill this apiErrorCallback....